_posts/2022-10-24-public-jenkins.markdown

```diff
@@ -0,0 +1,53 @@
+
+The build system is deployed using infrastructure as code that uses AWS CDK. The build infrastructure is completely reproducible and consumes [opensearch-ci](https://github.com/opensearch-project/opensearch-ci) repository as its source. The[readme.md](https://github.com/opensearch-project/opensearch-ci/blob/main/README.md) file in [opensearch-ci](https://github.com/opensearch-project/opensearch-ci) repository provides the instructions to reproduce this infrastructure. External contributors can contribute to the build ecosystem by raising the PR on [this](https://github.com/opensearch-project/opensearch-ci) GitHub repository for any future enhancements, bug fixes and new features. 
+
+## Motivation
+
+Before now, setting up the test and build infrastructure was a tedious process that required a lot of manual intervention and development effort to make changes and updates. Also, the infrastructure used for building and releasing the artifacts was not reproducible, and teams were finding it hard to track the status of builds due to limited visibility into the build process. This created discrepancies between the development team in terms of building, testing, and debugging the artifact builds during every release. 
+
+The manual processes also added additional overhead for the development teams to create, own and maintain their own automation build process, mimicking our process. The new system automates all the manual steps involved in creating, managing, maintaining and monitoring the DevOps stack for OpenSearch distribution builds to improve reproducibility along with developer and release velocity.
+
+### OpenSearch CI / CD infrastructure design:
+
+![Figure 1: Jenkins Infrastructure Overview]({{ site.baseurl }}assets/media/blog-images/2022-09-19-public-jenkins.markdown/jenkins.png){: .img-fluid }**Figure 1**: Jenkins Infrastructure Overview
+
+### Overview of the Jenkins workflow(s):
+
+All the workflows are managed via source controlled code located in the [GitHub Repository](https://github.com/opensearch-project/opensearch-build/tree/main/jenkins).
```

Comment:
```suggestion
All the workflows are managed through source-controlled code located in the [GitHub repository](https://github.com/opensearch-project/opensearch-build/tree/main/jenkins):
```

Reply:
We should name the repository rather than just referring to "the" repository. Which one?

Reply:
The "Github repository" hyperlink is the repository